test/persistence.jl

```diff
@@ -53,6 +53,11 @@ close(outread)
 close(outwrite)
 redirect_stdout(origSTDOUT)
 
+@test writegraphml("test1.graphml", "g3", g3) == 1
+@test writegraphml("test1.graphml", "g4", g4) == 1
+rm("test1.graphml")
```

Comment:
Could you use temp files here instead of hardcoded filenames? You can use [mktemp](http://docs.julialang.org/en/release-0.3/stdlib/file/#Base.mktemp) for this.